packages/web/modals/base.tsx

```diff
@@ -36,6 +37,11 @@ export const ModalBase: FunctionComponent<ModalBaseProps> = ({
 }) => {
   const { isMobile } = useWindowSize();
 
+  const bodyOpenClassNames = classNames("overflow-hidden", bodyOpenClassName);
+  useUnmount(() => {
+    document.body.classList.remove(bodyOpenClassNames);
+  });
```

Comment:
`ReactModal` should be doing this internally, but it isn't working as expected in the new bridge transfer.  I suspect this issue is tied to how we unmount the transfer modal on the assets page

Reply:
I also found bugginess with react modal. Maybe react 18 or a new version of react modal is the cause. I found unmounting and mounting modals (by conditionally rendering the whole modal) instead of using isOpen is causing issues, such as the screen not being scrollable. This is not great as well as some modals should have their state thrown out when closed.

Reply:
This is the same issue I had. I am reluctant to increase to version to avoid breaking other UI parts